test/docker_tests/test_ldap_ui.py

```
@@ -0,0 +1,247 @@
+        )
+
+        container_name = f"{project_name}-netalertx-1"
+        ldap_container = f"{project_name}-ldap-1"
+
+        # Connect the test container to the compose network to access the internal IPs directly
+        # The test container name is usually 'netalertx-test-container' or we can find it via hostname
+        test_container_id = socket.gethostname()
+        network_name = f"{project_name}_default"
+        print(f"Connecting test container {test_container_id} to network {network_name}...")
+        try:
+            subprocess.run(["docker", "network", "connect", network_name, test_container_id], check=False)
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)